src/lib/middlewares/appsI18n.js

```diff
@@ -1,4 +1,4 @@
-import { extend as extendI18n } from 'cozy-ui/react/I18n'
+import { extend as extendI18n } from 'cozy-ui/transpiled/react/I18n'
```

Comment:
So we can remove those loaders, right ? https://github.com/cozy/cozy-bar/blob/master/config/webpack.config.inline-styles.js#L39

Reply:
Reordered the commit that removed those lines and squashed it.